### DISCUSSION TITLE
Adjustable brightness for automatic keyboard dimming (RGB Sleep Mode)

### DISCUSSION THREAD

**Discussion by SymbolKlar** [WootingKb/wootility-issues#244] [Ideas]
It would be useful to be able to adjust the brightness the keyboard dims itself to when idle.

**Comment by joshmiller-hil** [comment]
The ultimate body-shaping secret, enhance your curves and boost confidence. Designed for comfort and support, they're your perfect companion for a slimmer, more confident you. [https://fajas-colombianas.info/](https://fajas-colombianas.info/
)
